code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -289,3 +289,12 @@
 	Neutral: Corn Oil, Ryetalyn, Tonio, Space Drugs
 	Bad: Mutagen, Polytrinic Acid, Lexorin, Zombie Powder
 	*/
+
+/obj/item/weapon/reagent_containers/pill/nanobot
+	name = "Nanobot pill"
+	desc = "Experimental medication."
+	icon_state = "pill7" //grey oblong
+
+	New()
```

Comment:
I have full faith in their abilities to put others work on me so I can add something for others.